package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "tutorials",
+  "version": "1.0.0",
+  "description": "Interledger tutorials",
+  "main": "pay.js",
+  "dependencies": {
+    "ilp-packet": "^1.3.0",
+    "ilp-plugin-payment-channel-framework": "github:interledgerjs/ilp-plugin-payment-channel-framework#bs-clp",
+    "ilp-plugin-xrp-escrow": "github:michielbdejong/ilp-plugin-xrp-escrow#72e4178",
```

Comment:
Why is this using a fork of the normal plugin?

Reply:
mainly because of https://github.com/ripple/ilp-plugin-xrp-escrow/pull/18

Reply:
That should get merged, but it also doesn't seem necessary for the basic tutorial, no?